styles.css

```diff
@@ -29,11 +29,27 @@ body {
   text-align: center;
   color: #f5f5f5;
   margin: 0;
-  overflow: none;
+  overflow-y: scroll;
+  overflow-x: hidden;
+  scroll-behavior: smooth;
```

Comment:
What's the effect of this property?

Reply:
overflow-x: hidden; is to get rid of the weird horizontal scroll.
overflow-y: scroll; is so the scrollbar will show on desktop even when there is no scroll available.
scroll-behavior: smooth; I don't think does anything in this context, it stayed from my experiments with locking the scroll.